kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/pom.xml

```diff
@@ -326,6 +327,7 @@
         <configuration>
           <container>
             <systemProperties>
+              <org.kie.server.bypass.auth.user>true</org.kie.server.bypass.auth.user>
```

Comment:
Same here. Why is the property specified in the integration test plugin?

Reply:
same as with cargo. this is for enabling client bypass client